### DISCUSSION TITLE
Does the execute method in Java API accept just the same format arguments as Launching as javaagent?

### DISCUSSION THREAD

**Discussion by Winson-Huang** [async-profiler/async-profiler#790] [Q&A] [answered]
Does the [execute method](https://github.com/async-profiler/async-profiler/blob/v2.9/src/api/one/profiler/AsyncProfiler.java#L193) in Java API accept just the same format arguments as [Launching as javaagent](https://github.com/async-profiler/async-profiler/tree/v2.9#launching-as-an-agent)?

**Comment by apangin** [comment, marked as answer]
Yes, the format and the arguments are the same.
